src/features/lessons/list-of-lessons/list-of-lessons.js

```diff
@@ -20,14 +21,18 @@ export const ListOfLessons = () => {
   const { data, isLoading } = (currentUser.role === 2) ? mentorsLessons : allLessons;
 
   const [searchLessonsThemeValue, setSearchLessonsThemeValue] = useState('');
+  const [rawLessonsList, setRawLessonsList] = useState([]);
   const [filteredLessonsList, setFilteredLessonsList] = useState([]);
   const [visibleLessonsList, setVisibleLessonsList] = useState([]);
-  const [searchLessonsDateValue, setSearchLessonsDateValue] = useState('');
+  const [filterStartDate, setFilterStartDate] = useState();
+  const [filterEndDate, setFilterEndDate] = useState();
+  const [startDateFilterBorder, setStartFilterDateBigger] = useState({
```

Comment:
The name of func in useState has to be like this:  set+name of state.
So if state = startDateFilterBorder, func name is setStartDateFilterBorder

Reply:
It is named that way.

Reply:
fixed.